### PR TITLE
LPS-116987

### PR DESCRIPTION
**Pull request by RitaSilveira**
JIRA Ticket:
LPS-136987
LPS-134749

Motivation:
Access the most relevant information through the side panel

### CODE REVIEW

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/LPS-134749, cc: @CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -1788,4 +1788,109 @@ definition {
...
+			Click(locator1 = "ContentDashboard#EDIT_OPTION");
+
+			CKEditor.addContentInWebContent(
+				content = "Web Content Content",
+				fieldLabel = "content");
+
+			Click(locator1 = "Button#SAVE_AS_DRAFT");
+			
+			Click(locator1 = "Icon#BACK");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: I think all these Click lines should be in a macro, like saveAsDraft(); or something like that

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 2]
```diff
@@ -1788,4 +1788,109 @@ definition {
...
+					portlet = "Content Dashboard");
+
+			Click.clickNoMouseOver(locator1 = "ContentDashboard#ROW_PANEL_WC");
+
+			Click(locator1 = "ContentDashboard#ICON_INFO_LISTED_MODIFIED_DATE");
+
+            Click(locator1 = "ContentDashboard#PANEL_INFO_VIEW");
+
+          
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: I think all these Click lines should be in a macro, like checkLanguage(); or something like that

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:forward

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**